corehq/apps/custom_data_fields/models.py

```diff
@@ -27,7 +28,7 @@ def validate_reserved_words(slug):
     if slug in SYSTEM_FIELDS:
         raise ValidationError(_('You may not use "{}" as a field name').format(slug))
     for prefix in [SYSTEM_PREFIX, 'xml']:
-        if slug and slug.startswith(prefix):
+        if slug and slug.startswith(prefix) and slug != GEO_POINT_CASE_PROPERTY:
```

Comment:
oh this is annoying, and we can't change the name now.
How come this was not an issue so far?

Reply:
I think thus far most of the testing has been done on cases and not users. As far as I know, the geospatial functionality isn't being used yet (aside from testing) so I think we could change this to something else and not need this extra validation.

Reply:
Yes please!!

Reply:
[e544203](https://github.com/dimagi/commcare-hq/pull/33420/commits/e544203876668d6d4ef75a7d28f904f2cde48fac)